code/game/machinery/autolathe/autolathe.dm

```diff
@@ -627,6 +659,11 @@
 
 	return disk.find_files_by_type(/datum/computer_file/binary/design)
 
+/obj/machinery/autolathe/proc/icon_off()
+	if(stat & NOPOWER)
+		return TRUE
+	return FALSE
```

Comment:
```suggestion
	return (stat & NOPOWER)
```

How that proc related to the icon actually? May be it should be replaced with somekind of `is_powered()` proc.

Reply:
Same problem as above.